tests_integration/test_procedure.py

```diff
@@ -0,0 +1,110 @@
+from __future__ import annotations
+
+import pytest
+
+from tests_integration.snowflake_connector import snowflake_session, test_database
+from tests_integration.testing_utils.naming_utils import object_name_provider
+from tests_integration.testing_utils.snowpark_utils import (
+    SnowparkTestSteps,
+    SnowparkTestSetup,
+    TestType,
+)
+from tests_integration.testing_utils.sql_utils import sql_test_helper
+from tests_integration.testing_utils.working_directory_utils import (
+    temporary_working_directory,
+)
+
+
+@pytest.mark.integration
+def test_snowpark_procedure_flow(_test_steps):
+    _test_steps.assert_that_no_entities_are_in_snowflake()
```

Comment:
what entities? There will be entities like accounts, roles. Should we be more explicite?

Reply:
I would suggest `_assert_no_procedures_in_test_schema`

Reply:
What about functions then? This test step is used for both